src/registered_functions.jl

```diff
@@ -117,7 +117,23 @@ expand_registered_functions(expr)
 Takes an expression, and expands registered function expressions. E.g. `mm(X,v,K)` is replaced with v*X/(X+K). Currently supported functions: `mm`, `mmr`, `hill`, `hillr`, and `hill`.
 """
 function expand_registered_functions(expr)
-    iscall(expr) || return expr
+    if hasnode(is_catalyst_function, expr)
+        expr = replacenode(expr, expand_catalyst_function)
+    end
+    return expr
+end
+
+# Checks whether an expression corresponds to a catalyst function call (e.g. `mm(X,v,K)`).
+function is_catalyst_function(expr)
+    iscall(expr) || (return false)
+    return operation(expr) in [
+        Catalyst.mm, Catalyst.mmr, Catalyst.hill, Catalyst.hillr, Catalyst.hillar]
```

Comment:
Use a tuple instead of vector to avoid allocations.

Reply:
This could also just be a global `const` variable.

Reply:
good point, yes, I am turning it to a const